conf/.htaccess.assets.cache.control

```diff
@@ -0,0 +1,10 @@
+# BEGIN core-standards:assets-cache-control
+<IfModule mod_headers.c>
+  # Browser Caching using Cache-Control headers.
+  <If "%{HTTP_HOST} !~ /local|nest/">
+    <FilesMatch "\.(?:css|cur|eot|gif|gz|htc|ico|jpe?g|js|mp4|ogg|ogv|otf|png|svgz?|ttf|webm|woff2?).*[?&]ver=">
+      Header set Cache-Control "max-age=31536000, public"
```

Comment:
The cacheability is normally specified first, because it can make all other parameters obsolete
```suggestion
      Header set Cache-Control "public, max-age=31536000"
```
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control